components/usage/pkg/db/cost_center_test.go

```diff
@@ -288,7 +288,7 @@ func TestSaveCostCenterMovedToStripe(t *testing.T) {
 	teamCC, err = mnr.UpdateCostCenter(context.Background(), teamCC)
 	require.NoError(t, err)
 	require.Equal(t, db.CostCenter_Stripe, teamCC.BillingStrategy)
-	require.Equal(t, db.VarcharTime{}, teamCC.NextBillingTime)
+	require.Equal(t, teamCC.CreationTime.Time().AddDate(0, 1, 0).Truncate(time.Second), teamCC.NextBillingTime.Time().Truncate(time.Second))
```

Comment:
I copied this from two lines below, assuming that there might be a random delay between CreationTime and the actual billing cycle start.

Happy to remove any truncation that doesn't make sense though.

Reply:
I see. In general, I'd only expect the truncation to be there if the implementation also truncates. This is because it can lead to MS level discrepancy and the test is then falsely validating something.

If needed, there's a `require.Delta` or similar methods which can help you check it's within a particular delta of the expected value

Reply:
While rebasing away the merge conflicts, I've also attempted to remove the truncation. Let's see if the test still passes.

Reply:
Seems to work. 🎉